### ISSUE TITLE
Disabled buttons do not aknowledge disabled state or provide disabled state image.

### ISSUE THREAD

**Issue by AraHaan** [TruDan/SkinFramework#3]
Basically the above making the form caption buttons flicker from windows classic style to the skin style which is not entirely correct.

I am not familiar with the code currently to remody this though.

**Comment by CatGirlsAreLife** [comment]
Yeah, I also notice that if the Minimized or maximized buttons are disabled on the target form that the skins will ignore that setting and still display and act like they are not disabled. Also if both are disabled in the target form they still display as well. Not to mention the help button is not even considered in the implementation at all which is a bad move. Also it would be nice if a user using the SkinManager would actually be able to disable centered text as it might not work on some application designs. Other than that I see not much of any other issue other than usage in Modal Dialogs seems to not theme the flicker if the nc like it does in office or visual studio windows. Not sure how to bypass that issue either.

**Comment by AraHaan** [comment]
@TruDan uh are you still working on this? I need some help with fixing this issue on my fork. I also would like some help with adding support for the help button too.
